chain/network/src/peer_manager.rs

```diff
@@ -369,13 +446,44 @@ impl PeerManagerActor {
             },
         );
 
-        self.process_edges(ctx, vec![new_edge.clone()]);
+        self.add_edges_to_routing_table(ctx, vec![new_edge.clone()]);
+
+        let wait_for_sync = 1;
```

Comment:
This constant already existed in the code without comments, I moved it up. I suspect wait 1 second to finish setting up connecting in order to avoid race conditations.